_posts/2023-02-15-my-first-patch-set.md

```diff
@@ -0,0 +1,153 @@
+---
+layout: post
+title:  "My First Patch Set"
+date:   2023-02-15 11:57:26 -0300
+categories: jekyll update
```

Comment:
It is good to create different categories so that you can write about other stuff in the future.